tests/integration/tier1/destructive/proxy-conversion/test_proxy_conversion.py

```diff
@@ -0,0 +1,141 @@
+import socket
+
+import pytest
+
+from conftest import SYSTEM_RELEASE_ENV, TEST_VARS
+
+
+def setup_proxy(shell):
+    """
+    Restrict the system internet connection only to a forward proxy server.
+    This is done by setting up a firewall setting on the machine.
+    Also for conversion to work it is necessary to enabled connection to specific
+    URLs directly from the machine.
+    """
+
+    # Make sure the firewalld is installed, enabled and running
+    shell("yum install -y firewalld")
+    shell("systemctl enable --now firewalld")
+
+    # As mentioned in the C2R documentation (https://url.corp.redhat.com/6d22a76)
+    # the following hostnames needs to be reachable
+    allowed_urls = [
+        "ftp.redhat.com",
+        "cdn-ubi.redhat.com",
+        "cdn.redhat.com",
+        "cdn-public.redhat.com",
+        "subscription.rhsm.redhat.com",
+        "static.redhat.com",
+        "cert.console.redhat.com",
+    ]
+    for url in allowed_urls:
+        ip_address = socket.gethostbyname(url)
+        assert (
+            shell(
+                f"firewall-cmd --permanent --direct --add-rule ipv4 filter OUTPUT 0 -p tcp -m tcp -d {ip_address} --dport=443 -j ACCEPT"
+            ).returncode
+            == 0
+        )
+        assert (
+            shell(
+                f"firewall-cmd --permanent --direct --add-rule ipv4 filter OUTPUT 0 -p tcp -m tcp -d {ip_address} --dport=80 -j ACCEPT"
+            ).returncode
+            == 0
+        )
```

Comment:
We can't pass --dport twice in the same command? That's a shame :(

Reply:
Yeah, I tried that, but it did not work.. /: